src/Sentry.AspNetCore.Grpc/Sentry.AspNetCore.Grpc.csproj

```diff
@@ -0,0 +1,21 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFrameworks>netcoreapp3.0;net5.0</TargetFrameworks>
```

Comment:
Do we really need net5.0 here? Wouldn't netcoreapp3.0 be enough?

Reply:
Probably should be good with just netcoreapp3.0. I'll change it.